README.md

```diff
@@ -55,7 +55,7 @@ installing the `ocne-catalog` chart.
 | Prometheus Adapter                                  | prometheus-adapter          | 0.10.0                     |
 | Prometheus Node Exporter                            | prometheus-node-exporter    | 1.6.1                      |
 | Rook                                                | rook                        | 1.12.3                     |
-| Tigera Operator with Calico 3.27                    | tigera-operator             | 1.32.4                     |
```

Comment:
If the `tigera-operator-1.32.4` chart is going to be retained, then please include it as well like `1.32.12<br>1.32.4`.